litestar/contrib/htmx/_utils.py

```diff
@@ -1,12 +1,22 @@
 from __future__ import annotations
 
-from enum import Enum
-from typing import TYPE_CHECKING, Any, Callable, cast
-from urllib.parse import quote
+from typing import TYPE_CHECKING
 
-from litestar.exceptions import ImproperlyConfiguredException
-from litestar.serialization import encode_json
+from litestar.utils import warn_deprecation
 
+if TYPE_CHECKING:
+    from litestar_htmx._utils import (  # noqa: TCH004
```

Comment:
While this is something we control, is it good practice to import a third party "private" module inside Litestar's private module?

Reply:
No, but I figured these methods may be used enough for us to have to have it re-exported.